### PR TITLE
Parse and populate `arg_bucket`-s

### PR DESCRIPTION
**Pull request by uvsmtid-gs**

### CODE REVIEW

File: src/argrelay/handler_response/DescribeLineArgsClientResponseHandler.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def render_result(
                 )
             else:
                 print(
-                    f"{TermColor.unconsumed_token.value}{interp_result.all_tokens[i]}{TermColor.reset_style.value}",
+                    f"{TermColor.remaining_token.value}{interp_result.all_tokens[i]}{TermColor.reset_style.value}",
```
Comment (uvsmtid) [inline src/argrelay/handler_response/DescribeLineArgsClientResponseHandler.py:58]: Consider highligting excluded tokens.

File: src/argrelay/runtime_context/InterpContext.py [hunk 2]
```diff
@@ -91,9 +119,10 @@ class InterpContext:
     """
 
     def __post_init__(self):
-        self.unconsumed_tokens = self._init_unconsumed_tokens()
+        self._init_consumption_state()
 
-    def _init_unconsumed_tokens(self):
+    # TODO: delete this - it is re-implemented in `_init_consumption_state`:
```
Comment (uvsmtid) [inline src/argrelay/runtime_context/InterpContext.py:124]: Clean up in this MR

File: src/argrelay/runtime_context/InterpContext.py [hunk 3]
```diff
@@ -91,23 +114,70 @@ class InterpContext:
     """
 
     def __post_init__(self):
-        self.unconsumed_tokens = self._init_unconsumed_tokens()
+        self._init_consumption_state()
```
Comment (uvsmtid) [inline src/argrelay/runtime_context/InterpContext.py:117]: Rename or inline?

Comment (uvsmtid-gs) [reply src/argrelay/runtime_context/InterpContext.py]: Done.

File: src/argrelay/test_infra/EnvMockBuilder.py [hunk 4]
```diff
@@ -939,23 +939,26 @@ def do_reset_local_server():
...
     return ParsedContext.from_instance(
         default_test_input_context(
             command_line,
             cursor_cpos,
+            comp_type,
         ),
     )
 
-
+# TODO: Rename to `default_test_call_context`, search for "input_context" to rename other places:
```
Comment (uvsmtid) [inline src/argrelay/test_infra/EnvMockBuilder.py:952]: Do in this MR.

File: setup.py [hunk 5]
```diff
@@ -36,7 +36,7 @@ def list_dir(
 setuptools.setup(
     name = "argrelay",
     # See `docs/dev_notes/version_format.md`:
-    version = "0.6.11",
+    version = "0.6.12.dev1",
```
Comment (uvsmtid) [inline setup.py:39]: Merge with non-dev version.

File: src/argrelay/test_infra/EnvMockBuilder.py (hunk 4 above)
Comment (uvsmtid-gs) [reply src/argrelay/test_infra/EnvMockBuilder.py]: Done.

File: setup.py (hunk 5 above)
Comment (uvsmtid-gs) [reply setup.py]: Done.

**Review by uvsmtid — DISMISSED** [review]

**Review by uvsmtid — DISMISSED** [review]

**Review by uvsmtid — APPROVED** [review]

**Pull request merged**